lisa/sut_orchestrator/baremetal/cluster/idrac.py

```diff
@@ -80,19 +107,67 @@ def type_schema(cls) -> Type[schema.TypedSchema]:
     def get_start_stop(self) -> Type[features.StartStop]:
         return IdracStartStop
 
+    def get_serial_console(self) -> Type[features.SerialConsole]:
+        return IdracSerialConsole
+
     def deploy(self, environment: Environment) -> Any:
         self.login()
         self._eject_virtual_media()
-        self._change_boot_order_once("VCD-DVD")
         assert self.client.iso_http_url, "iso_http_url is required for idrac client"
+        self._change_boot_order_once("VCD-DVD")
         if self.get_power_state() == "Off":
             self._log.debug("System is already off.")
         else:
-            self.reset("GracefulShutdown")
+            self.reset("ForceOff")
         self._insert_virtual_media(self.client.iso_http_url)
+        check_till_timeout(
```

Comment:
insert media after it's off. I'm not sure whether weird behaviors when inserting and turn off happening together.